foundry.toml

```diff
@@ -1,19 +1,12 @@
 [profile.default]
-src = 'contracts'
-out = 'foundry_artifacts'
+src = "ready_src"
+out = "foundry_artifacts"
 libs = ["node_modules", "lib"]
 gas_reports = ["*"]
 memory_limit = 11149934592
-fs_permissions = [{access = "read", path = "./test/uarch-bin/"}, {access = "read", path = "./test/uarch-log/"}]
-remappings = [
-    '@cartesi/=node_modules/@cartesi/',
-    '@ensdomains/=node_modules/@ensdomains/',
-    '@openzeppelin/=node_modules/@openzeppelin/',
-    'eth-gas-reporter/=node_modules/eth-gas-reporter/',
-    'hardhat-deploy/=node_modules/hardhat-deploy/',
-    'hardhat/=node_modules/hardhat/',
-]
+fs_permissions = [{access = "read", path = "./test/uarch-log/"}, {access = "read", path = "./test/uarch-bin/"}]
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
-# To run forge tests, install forge cli and run below command
-# forge install foundry-rs/forge-std --no-commit
+[fmt]
+line_length = 80
+ignore = ["test/UArchReplay_*", "src/"]
```

Comment:
Any reason for skipping `src/`? Like, I believe if we make all files valid Solidity, `fmt` should work.

Reply:
I thought the function conflicts in `production` and `mock` can fail the `fmt` command. I was wrong, my bad. Will not sKip `src`